pages/api/links/[id]/archive.ts

```diff
@@ -8,6 +8,24 @@ export default async function handle(
   req: NextApiRequest,
   res: NextApiResponse
 ) {
+  if (req.method == "GET") {
```

Comment:
Why are we creating a new `GET` endpoint? to resolve this issue we just need to get another parameter from existing `GET` which you can trace it by finding API call. Please test it locally before rushing to the PR.

Reply:
Thanks for the feedback. I'm on it.

Reply:
I found that one GET endpoint was already available in the `api/links` route. I modified that GET endpoint so that it sends the `isArchived` property of the link. Can you please give it a look and let me know if further changes are required?